src/data/ACTIONS/root/NIN.ts

```diff
@@ -261,7 +261,7 @@ export const NIN = ensureActions({
 		name: 'Ten Chi Jin',
 		icon: 'https://xivapi.com/i/002000/002922.png',
 		onGcd: false,
-		cooldown: 100,
+		cooldown: 120,
```

Comment:
This (and a bunch of other NIN adjustments) were technically in 5.1, so maybe use layers here? The complete mudra rework in 5.1 prob complicates stuff tho.

Reply:
I didn't bother because NIN isn't supported by xiva pre-5.1 anyway. The reworked changed a *lot* of stuff.